jupyterhub/values.yaml

```diff
@@ -144,6 +143,10 @@ proxy:
     loadBalancerIP:
     loadBalancerSourceRanges: []
   chp:
+    containerSecurityContext:
+      runAsUser: 65534
```

Comment:
Is this the 'nobody' user? If so, can we explicitly list that?

Reply:
Yepp this is the nobody user, but I think due to https://github.com/jupyterhub/configurable-http-proxy/pull/211, we should stick with the numerical value to identify the user.

Reply:
oh yeah, makes sense! I just want us to add a comment or similar stating what this number is - otherwise it's a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming))

Reply:
Deal! Resolved by 4c0bacf!